configs/README.md

```diff
@@ -0,0 +1,31 @@
+# Configuration files
+
+Directories `data` and `representation` contain configuration files required to reproduce results of the study.
+Directory `models` contains an exemplary configuration file and a script to generate all possible hyperparameter configurations.
+
+All configuration files are parsed by methods implemented in `graphrepr.config`.
```

Comment:
```suggestion
All configuration files are parsed by the methods implemented in `graphrepr/config.py`.
```

Reply:
why this suggestion: it wasn't clear that you mean a python package and not a config file called graphrepr.config